server/api/datasets/[datasetid]/agreements.get.ts

```diff
@@ -108,6 +123,12 @@ export default defineEventHandler(async (event) => {
     data: agreements.map((item) => ({
       id: item.id,
       name: `${item.download_user_details.given_name} ${item.download_user_details.family_name}`,
+      affiliation: item.download_user_details.affiliation,
+      approval_date: dayjs
+        .unix(Number(BigInt(item.download_request[0].approval.updated_on)))
+        .format("MMM D, YYYY"),
+      dataset_id: item.dataset_id,
```

Comment:
I'm pretty sure I tested using the version_title here as well and nothing was returned. I may try once more just in case I spaced adding to the data response

Reply:
Yep - this was the problem. I'll push an update with the fix.